src/server/main.js

```diff
@@ -63,9 +70,25 @@ io.on('connect', (socket) => {
    returns JWT
    */
   socket.on('login', async (user) => {
-    const login = await authentication.login(user);
+    const token = await authentication.login(user);
```

Comment:
this `token` is what should be tested. It could be a `String` if there was an error

Reply:
I implemented the changes in the comment immediately below this one

In this case, even if the token is an error ```String```, when calling 
```socketUser = authentication.verifyToken(token);```, this verification will return a ```false```.

The output of ```socketUser = authentication.verifyToken(token);``` is checked, if it's ```false```, then it will return an error to the user, otherwise it will return the token